lib/prometheus_telemetry/metrics/ecto.ex

```diff
@@ -35,6 +35,14 @@ if PrometheusTelemetry.Utils.app_loaded?(:ecto) do
     end
 
     def metrics(prefix) do
+      if is_atom(prefix) and Code.ensure_loaded?(prefix) do
```

Comment:
This doesn't quite work because with Ecto you can specify a singular prefix that spans multiple repos, for example at blitz we use `metrics(:blitz)` and all of our repos publish under the same prefix!

Reply:
Ah, I see. But it wouldn't raise because it's not a module, right? I get the same result as main from `metrics(:blitz)`: 4 distributions with event_name `[:blitz, :query]`. What's the expected result?